Add a RunKit example to Aphrodite

xymostech opened the pull request:
Summary: RunKit (previously TonicDev) lets you add a custom example on
the package page linked to from npmjs.com:
https://discuss.runkit.com/t/customizing-your-npm-packages-page-on-tonic/23

This adds such an example!

Closes #140

Test Plan:
 - Publish this as the new package, `aphrodite-test`
 - See that the runkit integration works :)

xymostech commented:
https://runkit.com/npm/aphrodite-test

jlfwong commented on examples/runkit.js:
```diff
@@ -0,0 +1,33 @@
+
+// Make a component that generates some styles and returns some HTML.
+function render() {
+    const {StyleSheet, css} = require("aphrodite/no-important");
```
Seems weird to use the non-default option in the example

xymostech replied:
:P If a console statement is too long in RunKit, it adds an ellipse, which I thought was annoying, so I went with this to fix that. I'm happy to undo that though!

![image](https://cloud.githubusercontent.com/assets/217900/18730137/90d1e01e-8009-11e6-8001-099c626449fa.png)

jlfwong replied:
Hehe, sad, but okay!

xymostech commented:
> Did you mean to check in dist/aphrodite.js? Looks like those changes have nothing to do with this review.

updated the pr, I had a git pre-commit hook which automatically built the files from back when we always committed built files. I removed it now :P

jlfwong commented:
LGTM!

xymostech commented:
Can you accept the review? I can push it anyway but I want to see what it looks like when you do :)

xymostech commented:
Also, I brought this up a bit ago but now that we have new things to land, I'll bring it up again: Do we want to move to version 1.0 next time we release? Semver says

> If your software is being used in production, it should probably already be 1.0.0.

and Aphrodite is very in production.

kentcdodds commented:
Yes please :)

jlfwong commented:
@xymostech Was going to suggest we make 1.0.0 the version with plugins, though perhaps it's weird to do the bump to 1.0.0 when introducing a somewhat experimental feature? I don't have strong feels, and would be happy for us to bump immediately if you think that's more appropriate.

xymostech commented:
Oh yeah, we can do it after that lands. Just making sure you're on board with doing it sometime soon :)
